assessment.md

```diff
@@ -119,29 +119,47 @@ TODO: add description for each assessment level for this skill.
 ### Continuous Integration with Github Actions
 
 * Setup and configuration of GitHub Actions CI
-
-TODO: add description for each assessment level for this skill.
+  * None = Does not use Github Actions CI or have it set up on their repository.
+  * Inadequate = Set up Github Actions to a workflow that doesn't make sense with the program and its contributors or in a way that isn't functionally committed to the repository.
+  * Adequate = Properly set up a Github Actions workflow in a way that is usable to collaborators.
+  * Good = Set up a Github Actions workflow that streamlines the workflow process and makes process consistent among devices that have different installations.
+  * Excellent = Streamlines all major angles including testing and general communication using Github Actions.
 
 * Performing secure releases of project deliverables
-
-TODO: add description for each assessment level for this skill.
+  * None = Does not set up any security features on project deliverables.
+  * Inadequate = Sets up some security features on project deliverables, but project information is still easily compromisable.
+  * Adequate = Properly sets up all appropriate security features on project deliverables.
+  * Good = Properly sets up all appropriate security features on project deliverables in way that is well-suited for given project deliverables.
+  * Excellent = Properly sets up all appropriate security features on project deliverables in way that is best-suited for given project deliverables, and actively monitors project for potential security threats or improvements.
 
 * Creating third-party integrations with GitHub Actions CI (e.g., codecov.io)
-
-TODO: add description for each assessment level for this skill.
+  * None = Does not create third-party integrations with GitHub Actions CI.  
+  * Inadequate = Is able to occasionally integrate GitHub with a third party. Still lacks fundamental knowledge of how to do so consistently
+  * Adequate = Contributes third party integrations on a fairly consistent basis. Could use more knowledge on the subject, however.
+  * Good = Often times integrates third party software with Github. Very knowledgeable on how to do so.
+  * Excellent = Leads the way for the team when integrating Github with a third party. Can easily explain to the team how they did so.
 
 * Adding README badges that show status of GitHub Actions CI builds
-
-TODO: add description for each assessment level for this skill.
+  * None = Does not add README badges.
+  * Inadequate = Rarely adds README badges that show status of builds, never addressing any errors made.
+  * Adequate = Occasionally adds README badges that show status of builds, rarely addressing errors any errors made.
+  * Good = Frequently adds README badges that show status of builds, often addressing any errors made.
```

Comment:
This assessment is to some degree unrealistic. Often time, there are only very few badges that could be added to reflect the build status. "Frequently adds badges" would seem impossible. It would be best for the team to come up with other assessments on the skill of using README badges.